src/Semantic/Graph.hs

```diff
@@ -245,8 +244,8 @@ parsePackage parser project = do
     n = Data.Abstract.Evaluatable.name (projectName project) -- TODO: Confirm this is the right `name`.
 
 -- | Parse all files in a project into 'Module's.
-parseModules :: (Member Distribute sig, Member (Error SomeException) sig, Member Parse sig, Carrier sig m) => Parser term -> Project -> m [Module (Blob, term)]
-parseModules parser p@Project{..} = distributeFor (projectFiles p) (parseModule p parser)
+parseModules :: (Member Distribute sig, Member Parse sig, Carrier sig m) => Parser term -> Project -> m [Module (Blob, term)]
+parseModules parser p = distributeFor (projectBlobs p) (parseModule p parser)
```

Comment:
`distributeFor` doesn’t actually provide us any concurrency, but that’s an issue for a later time.